### PR TITLE
Add Cisco Proximity cask (latest).

### PR DESCRIPTION
**Pull request by rajiv**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/cisco-proximity.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  name 'Cisco Proximity'
+  homepage 'https://proximity.cisco.com/'
+  license :gratis
+
+  app 'Cisco Proximity.app'
+
+  uninstall signal: [
+                      ['TERM', 'com.cisco.experimental.Proximity'],
+                    ]
+  uninstall launchctl: 'com.cisco.proximity'
```
Comment (reitermarkus) [inline Casks/cisco-proximity.rb]: These should be in one `uninstall` stanza.

File: Casks/cisco-proximity.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  sha256 :no_check
+
+  url 'https://proximity.cisco.com/mac/Proximity.dmg'
+  name 'Cisco Proximity'
+  homepage 'https://proximity.cisco.com/'
+  license :gratis
+
+  app 'Cisco Proximity.app'
+
+  uninstall signal: [
```
Comment (jawshooah) [inline Casks/cisco-proximity.rb]: Did `uninstall :quit` not work?

Comment (rajiv) [inline Casks/cisco-proximity.rb]: `uninstall :quit` did not work. i tested it and the app did not quit.

File: Casks/cisco-proximity.rb (hunk 1 above)
Comment (rajiv) [inline Casks/cisco-proximity.rb]: interesting that `brew cask style` didnt complain. i will refactor.

**Comment by jawshooah** [comment]
Thanks!

**Pull request merged**